pkg/apis/management.cattle.io/v3/machine_types.go

```diff
@@ -400,9 +400,10 @@ type CloudCredential struct {
 }
 
 type CloudCredentialSpec struct {
-	DisplayName        string              `json:"displayName"`
-	Description        string              `json:"description,omitempty"`
-	S3CredentialConfig *S3CredentialConfig `json:"s3credentialConfig,omitempty"`
+	DisplayName             string                   `json:"displayName"`
+	Description             string                   `json:"description,omitempty"`
+	S3CredentialConfig      *S3CredentialConfig      `json:"s3credentialConfig,omitempty"`
+	AlibabaCredentialConfig *AlibabaCredentialConfig `json:"alibabacredentialConfig,omitempty"`
```

Comment:
```suggestion
	S3CredentialConfig      *S3CredentialConfig      `json:"s3credentialConfig,omitempty"`
	AlibabaCredentialConfig *AlibabaCredentialConfig `json:"alibabaCredentialConfig,omitempty"`
```

Reply:
the other credential schemas had the same format so kept it as it is.
s3 credentials also has the same pattern.